src/main/java/com/echobox/api/linkedin/types/videos/VideoDetails.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.types.videos;
+
+import com.echobox.api.linkedin.jsonmapper.LinkedIn;
+import com.echobox.api.linkedin.types.urn.URN;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class VideoDetails {
+  
+  public VideoDetails(String downloadUrl) {
+    this.downloadUrl = downloadUrl;
+  }
+  
+  @LinkedIn
+  private URN owner;
+  
+  @LinkedIn
+  private URN id;
+  
+  @LinkedIn
+  private Long duration;
+  
+  @LinkedIn
+  private Float aspectRatioWidth;
+  
+  @LinkedIn
+  private Float aspectRatioHeight;
+  
+  @LinkedIn
+  private String downloadUrl;
+  
+  @LinkedIn
+  private Long downloadUrlExpiresAt;
+  
+  @LinkedIn
+  private String status;
+  
+  @LinkedIn
+  private String thumbnail;
```

Comment:
Haven't added the `String captions` [field](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/videos-api?view=li-lms-2023-03&tabs=http#schema) as we don't support uploading captions but native video posts might contain it so not sure if we would like to store this?

```
captions - File identifier to retrieve the caption file. Present if initializeUploadRequest.uploadCaptions was true and a caption was successfully uploaded and processed.
```